src/reportSources/legacyFflogs/eventAdapter/prepullAction.ts

```diff
@@ -24,7 +24,7 @@ export class PrepullActionAdapterStep extends AdapterStep {
 		return {
 			...event,
 			type: 'action',
-			timestamp: this.pull.timestamp,
+			timestamp: this.pull.timestamp - 1,
```

Comment:
ah, thanks for adding this. i'm _tempted_ to suggest it should be in `base` or a new `consts` given we're sharing this between prepull action and status, but equally i'm not sure that the actual value is important - i guess, if it's important that the `-1` and `-2` are in that particular order, it should probably be shared.

Reply:
Seems like a good idea, maybe something like this in `base`?

```typescript
const PREPULL_OFFSETS = {
	STATUS_ACTIONS: -3,
	STATUS_APPLICATIONS: -2,
	PULL_ACTION: -1,
} as const
```

Reply:
sgtm